src/PanoImageRenderer/VideoLoader.js

```diff
@@ -71,14 +78,18 @@ export default class VideoLoader {
 
 	get() {
 		return new Promise((res, rej) => {
+			// on iOS safari, 'loadeddata' will not triggered unless the user hits play,
+			// so used 'loadedmetadata' instead.
+			const isIOS = Agent().os.name === "ios";
+			const thresholdReadyState = isIOS ? READY_STATUS.HAVE_METADATA : READY_STATUS.HAVE_CURRENT_DATA;
+			const thresholdEventName = isIOS ? "loadedmetadata" : "loadeddata";
+
```

Comment:
@happyhj Is it necessary to branch iOS code? 
If necessary then... how about to move this code on VideoLoader constructor?

Reply:
I removed conditional logic.  VideoLoader will use HAVE_METADATA ready state to determine the video resource is resolved.